src/components/ColumnTitle/ColumnFilter.tsx

```diff
@@ -145,7 +150,10 @@ export const ColumnFilter: FunctionComponent<ColumnFilterProps> = (props) => {
 
     const {t} = useTranslation()
 
-    const content = <div className={styles.content}>
+    const content = props.widgetMeta.type === FieldType.multivalue
+        ? visible &&
+        handlePopup(props.widgetMeta.popupBcName, props.bcName, props.widgetMeta.assocValueKey, props.widgetMeta.associateFieldKey)
```

Comment:
`handlePopup` is of `(...args) => void` type, so it does not return anything and can not be used as part of logical expression per se. So the only reason it is there is to call side effects - which is a thing that we definetly should try to avoid.
So we have: `visible && handlePopup(args)` will always result in undefined (or void in typescipt environment), as it really is just a `visible && undefined` call, so although it is not clear from the code the content is always `null` for multivalue fields.

If `props.onMultivalueAssocOpen` and `handleVisibleChange` should be called on component mount when field has  multivalue type and visible flag is set, then it is a side effect and should be described with `useEffect` hook:
```tsx
const isMultivalue = props.widgetMeta.type === FieldType.multivalue
const autoOpen = React.useEffect(() => {
    if (isMultivalue  && visible) {
        handlePopup()
    }
}, [dependencies])
// ...
const content = isMultivalue ? undefined : // ...
```

After that you also will not need this `content={content || null}`

Reply:
fixed with hooks